lib/compiler/passes/generate-bytecode.js

```diff
@@ -510,7 +584,15 @@ function generateBytecode(ast, options) {
       )
     );
   }
-
+  /**
+   *
+   * @param {PEG.ast.Expression|null} delimiterNode
+   * @param {number} expressionMatch
+   * @param {number[]} expressionCode
+   * @param {Context} context
+   * @param {number} offset
+   * @returns
```

Comment:
All of these with `@returns` need to have the correct type after, like `@returns {string}`